postgresql/CMakeLists.txt

```diff
@@ -2,10 +2,25 @@ project(userver-postgresql CXX)
 
 find_package(Boost REQUIRED regex)
 
+# We need libldap to statically link with libpq
+# There is no FindLdap.cmake and no package config files
+# for ldap library, so need to search for it by hand.
+find_library(LDAP_LIBRARY NAMES ldap)
+if(NOT LDAP_LIBRARY)
+  message(FATAL_ERROR "Failed to find libldap.so.\n"
+      "The linux system ldap installs shared objects with very ugly names, "
+      "so please install `libldap2-dev` package. "
+      "For Mac OS X please install `openldap`.")
+endif()
 
 option(USERVER_FEATURE_PATCH_LIBPQ "Apply patches to the libpq (add portals support)" ON)
 add_library(userver-libpq INTERFACE)
 if (USERVER_FEATURE_PATCH_LIBPQ)
+  # With Conan and libpq patching, use the system library as a workaround,
+  # because Conan's libpq does not expose internal APIs. Delete the
+  # Conan-generated Find*.cmake to reveal the cmake_generated version.
+  file(REMOVE "${CMAKE_BINARY_DIR}/FindPostgreSQL.cmake")
```

Comment:
Removed, I couldn't find a way to build `postgresql_extra` anyway :sweat_smile:

Reply:
Yep, I couldn't too because we need krb5 in cci https://github.com/conan-io/conan-center-index/issues/4102 so I think conan build will be without pq-extra